c10/util/Exception.h

```diff
@@ -412,6 +416,13 @@ inline std::string if_empty_then(std::string x, std::string y) {
   }()
 #endif
 
+#define TORCH_WARN_MAYBE(...) \
```

Comment:
I think we should just replace TORCH_WARN_ONCE with this.

Reply:
done